packages/docs/src/components/api/ApiTable.vue

```diff
@@ -3,12 +3,135 @@
     v-bind="$attrs"
     v-on="$listeners"
   >
-    <slot />
+    <thead>
+      <tr>
+        <th v-for="header in headers" :key="header">
+          <div class="text-capitalize">{{ header }}</div>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <template v-for="item in items">
+        <tr :key="item.name" :class="['regular-row', hasExtraRow(item) && 'has-extra-row']">
+          <td v-for="(header, i) in headers" :key="i">
+            <template v-if="header === 'name'">
+              <div class="font-weight-bold text-mono">{{ item[header] }}</div>
+            </template>
+            <template v-else-if="header === 'type' || header === 'signature'">
+              <div class="text-mono">{{ Array.isArray(item[header]) ? item[header].join(', ') : item[header] }}</div>
+            </template>
+            <template v-else-if="header === 'default'">
+              <code>{{ getDefaultValue(item) }}</code>
```

Comment:
we should probably make this markup for sass variables (especially for multi-line sass vars)
`<markup language="sass" :code="getDefaultValue(item)" />`